app/src/main/java/com/babylon/wallet/android/presentation/ui/composables/ChooseAccountContent.kt

```diff
@@ -58,28 +65,32 @@ fun ChooseAccountContent(
                 title = stringResource(id = R.string.empty),
                 onBackClick = onBackClick,
                 backIconType = if (showBackButton) BackIconType.Back else BackIconType.Close,
-                windowInsets = WindowInsets.statusBars
+                windowInsets = WindowInsets.statusBars,
             )
         },
         bottomBar = {
             BottomPrimaryButton(
-                onClick = onContinueClick,
-                enabled = isContinueButtonEnabled,
                 modifier = Modifier
                     .fillMaxWidth()
                     .navigationBarsPadding(),
```

Comment:
should be removed, right now when you scroll accounts you can see them scrolling below the continue button, can be seen on 4th screenshot you attached.

Reply:
I noticed this but I thought it will be fixed after merging [#1035](https://github.com/radixdlt/babylon-wallet-android/pull/1035). I'll make the change anyway, thank you!

Reply:
Exactly @jakub-rdx I would suggest after the release of 1.8.0 to merge first the #1035 and then take care of individual PRs case by case.